Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.13.1-alpine3.21

```

Comment:
The base image `python:3.13.1-alpine3.21` does not exist. Please update it to a stable and available image like `python:3.10-slim` to ensure the Docker image builds correctly.